README.md

```diff
@@ -29,20 +29,111 @@ The <b>Agglayer</b> (<i>Aggregation layer</i>) provides a common language for se
 
 </div>
 
-## Getting Started
+## Table of Contents
 
-Rust implementation of the Agglayer.
+- [Overview](#overview)
+- [Crate Directory](#crate-directory)
+- [Prerequisites](#prerequisites)
+  - [Succinct Prover Network](#succinct-prover-network)
+  - [Software Requirements](#software-requirements)
+  - [Hardware Requirements](#hardware-requirements)
+- [Installation](#installation)
+- [Running the Pessimistic Proof Test Suite](#running-the-pessimistic-proof-test-suite)
+- [Development](#development)
+- [Support](#support)
+- [Resources](#resources)
+- [License](#license)
 
-Agglayer is a web service that receives ZKPs from different CDK chains and checks the soundness of them before sending the ZKP to L1 for verification.
+## Overview
 
-To find out more about Polygon, visit the [official website](https://docs.polygon.technology/cdk/).
+AggLayer is the Rust-based service designed to (1) receive zero-knowledge proofs from AggLayer-connected chains, (2) verify their validity, and (3) send them to the L1 for final settlement. AggLayer replaces the previous Golang implementation. 
+
+To find out more about Polygon, visit the [official website.](https://docs.polygon.technology/cdk/)
 
 WARNING: This is a work in progress, and as such, all APIs and configuration are subject to change. The code is still being audited, so please contact the Polygon team if you would like to use it in production.
 
+## Crate Directory
+
+The crates and their functions within the AggLayer repo are as follows:
+
+| Crate                          | Description                                                                                                  |
+|--------------------------------|--------------------------------------------------------------------------------------------------------------|
+| agglayer-aggregator-notifier   | Contains certificate implementations for `certify` and `settled`, as well as `packer`, which handles the packing of an epoch |
+| agglayer-certificate-orchestrator | Manages the orchestration and handling of certificates; also handles `current_epoch`, which allows non-orchestrators to push a proven certificate |
```

Comment:
Nit - it would be nice to use end of sentence periods consistently for `Description` sentences.

Reply:
I'm gonna pass on this, more of a stylistic thing

Reply:
(apologies if I misunderstood, but I don't see any inconsistencies there